sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -271,4 +286,20 @@ protected void doStop() {
     @Override
     protected void doClose() {
     }
+
+    /**
+     * @throws SchemaUnknownException if the view was not found and no such schema exists
+     */
+    @Nullable
+    public String resolveView(TableIdent tableIdent) {
```

Comment:
why is not `throws SchemaUnknownException` in the signature?

Reply:
RuntimeExceptions shouldn't be added to the signature

Reply:
yep, ok didn't know ti's Runtime.